alkira/resource_alkira_connector_gcp_vpc.go

```diff
@@ -172,6 +172,12 @@ func resourceAlkiraConnectorGcpVpc() *schema.Resource {
 					"SMALL", "MEDIUM", "LARGE", "2LARGE",
 					"4LARGE", "5LARGE", "10LARGE", "20LARGE"}, false),
 			},
+			"customer_asn": {
+				Description: "A specific BGP ASN for the connector. This cannot be specified for connectors in `VNET_PEERING` mode. This field cannot be updated" +
+					"once the connector has been provisioned. The ASN can be any private ASN (64512 - 65534, 4200000000 - 4294967294) that is not used elsewhere in the network.",
```

Comment:
For here, it will be good to keep values in the code block of Markdown:

The value of the field could be any private ASN (`54512 - 65534`,  `4200000000 - 4294967294`)